app/uk/gov/hmrc/agentmappingfrontend/controllers/MappingController.scala

```diff
@@ -114,3 +105,17 @@ class MappingController @Inject()(override val messagesApi: MessagesApi,
     Future successful Ok(html.not_enrolled())
   }
 }
+
+object MappingController {
+
+  val mappingForm = Form(
+    mapping(
+      "arn" -> mapping(
+        "arn" -> arn
```

Comment:
im not sure whether the arn requires the second embedded "mapping" as the field has the same name "arn" arn"
whereas the utr has value for it.

Reply:
maybe not, but changing this is not a part of this story